frontend/src/app/supportAdmin/ManageUsers/AdminManageUser.tsx

```diff
@@ -210,79 +244,223 @@ export const AdminManageUser: React.FC = () => {
         ) {
-          setDisplayedError(userIdentityError);
-          setFoundUser(undefined);
+          setSearchState((prevState) => ({
+            ...prevState,
+            displayedError: userIdentityError,
+            foundUser: undefined,
+          }));
         } else if (error) {
-          setDisplayedError(genericError);
-          setFoundUser(undefined);
+          setSearchState((prevState) => ({
+            ...prevState,
+            displayedError: genericError,
+            foundUser: undefined,
+          }));
         } else {
-          setDisplayedError(undefined);
-          setFoundUser(data?.user as SettingsUser);
+          reset({
+            role: data?.user?.role || "USER",
+            organizationId: data?.user?.organization?.id,
+            facilityIds:
+              data?.user?.organization?.testingFacility.map(
+                (facility) => facility.id
+              ) || [],
+          });
+
+          setSearchState((prevState) => ({
+            ...prevState,
+            displayedError: undefined,
+            foundUser: data?.user as User,
+          }));
         }
       }
     );
   };
 
   const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    setSearchEmail(e.target.value);
+    setSearchState((prevState) => ({
+      ...prevState,
+      searchEmail: e.target.value?.trim(),
+    }));
   };
   const handleClearFilter = () => {
-    setSearchEmail("");
-    setDisplayedError(undefined);
-    setFoundUser(undefined);
+    setSearchState(initialState);
+    reset();
   };
 
+  const handleSearch = () => {
+    retrieveUser(searchEmail);
+  };
+
+  /**
+   * Organization access form setup
+   */
+  const {
+    control,
+    handleSubmit,
+    formState: { errors, isDirty },
+    register,
+    trigger,
+    reset,
+    setValue,
+  } = useForm<OrgAccessFormData>();
+
+  const formValues = useWatch({
+    control,
+    defaultValue: {
+      organizationId: "",
+      role: "USER",
+      facilityIds: [],
+    },
+  });
+
+  /**
+   * Fetch facilities
+   */
+  const [
+    queryGetFacilitiesByOrgId,
+    { data: facilitiesResponse, loading: loadingFacilities },
+  ] = useGetFacilitiesByOrgIdLazyQuery();
+
+  useEffect(() => {
+    if (formValues.organizationId) {
+      queryGetFacilitiesByOrgId({
+        fetchPolicy: "no-cache",
+        variables: {
+          orgId: formValues.organizationId,
+        },
+      });
+
+      if (formValues.organizationId !== foundUser?.organization?.id) {
+        setValue("facilityIds", []);
+      }
+    } else {
+      setValue("facilityIds", []);
+    }
+  }, [
+    queryGetFacilitiesByOrgId,
+    formValues.organizationId,
+    setValue,
+    foundUser?.organization?.id,
+  ]);
+
+  const facilityList = formValues.organizationId
+    ? facilitiesResponse?.organization?.facilities.map(
+        (facility) =>
+          ({ id: facility.id, name: facility.name } as UserFacilitySetting)
+      )
+    : [];
```

Comment:
For all facilities checkbox, it doesn't show up on the admin org page.
![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/45368936-c839-488a-8b34-f3c1df000f78)
![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/c89a4987-0a0b-4a71-9c6c-14848555bf52)

Reply:
The form in the organization admin page and the site admin page are not the same because the site admin page is using react-hook-forms and the org admin is still using the custom implementation. I did not migrate the org admin form because the scope of the ticket would get too big.

Reply:
Again with me not being clear about the issue 😓 . Sorry!!!
On the org admin facilities access section, if you select "Access all facilities" it won't appear once saved.


https://github.com/CDCgov/prime-simplereport/assets/10108172/5148c2be-8cbb-450f-8768-566d03c51b45